rollup.config.js

```diff
@@ -3,24 +3,42 @@ import postcss from 'rollup-plugin-postcss'
 
 import { default as importHTTP } from 'import-http/rollup'
 
-export default {
+const isProd = process.env.NODE_ENV === 'production'
+
+const devConfig = {
   input: 'app/index.js',
   output: {
     file: 'app/bundle.js',
-    format: 'es',
+    format: 'esm',
     sourcemap: 'inline',
   },
   plugins: [
-    resolve({
-      jsnext: true,
-    }),
+    resolve(),
     importHTTP(),
     postcss({
-      extract: false,
       inject:  false,
-    })
+    }),
   ],
   watch: {
     exclude: ['node_modules/**'],
   }
-}
\ No newline at end of file
+}
+
+const prodConfig = {
+  input: 'app/index.js',
+  output: {
+    file: 'dist/bundle.js',
+    format: 'esm',
+    sourcemap: true,
+  },
+  plugins: [
+    resolve(),
+    importHTTP(),
+    postcss({
+      extract: true,
+      minimize: true,
+    }),
+  ]
+}
+
+export default isProd ? prodConfig : devConfig
```

Comment:
cool, i like this pattern as well for prod builds. i need to do some tests to verify the output is as expected. thanks for doing this!